tools/mo/requirements_tf.txt

```diff
@@ -1,6 +1,7 @@
 # TensorFlow 1.x and 2.x are incompatible, use separate virtual environments for each version
 tensorflow~=1.15.5
-networkx~=2.5
+networkx~=2.5; python_version == "3.6"
```

Comment:
I think we should install ~=2.5 version for earlier python version despite we do not support it 3.5 or earlier versions.

Reply:
Yes, we should.
Updated, now we check python_version <= "3.6"